chain/network/src/peer_manager.rs

```diff
@@ -171,8 +198,18 @@ pub struct PeerManagerActor {
     txns_since_last_block: Arc<AtomicUsize>,
     pending_incoming_connections_counter: Arc<AtomicUsize>,
     peer_counter: Arc<AtomicUsize>,
+    peer_white_list: Arc<RwLock<HashSet<IpAddr>>>,
```

Comment:
Why do we need a lock here?

Reply:
We need it, because we don't have access to PeerActor in the place where we drop incoming connections.
To make things cleaner I moved this change to https://github.com/near/nearcore/pull/4825